syncer/db_test.go

```diff
@@ -275,7 +275,10 @@ func (s *testDBSuite) TestTimezone(c *C) {
 					continue
 				}
 
-				rowid := ev.Rows[0][0].(int32)
+				rowid, ok := ev.Rows[0][0].(int32)
```

Comment:
any concurrent case to run? or does not `resetBinlogSyncer` help to filter other cases?

Reply:
I revert this change (in https://github.com/pingcap/dm/pull/360/commits/aa3d4e77274bb56781582ba0fff369923e4edae2) and will split the unit test cases (with MySQL) into different containers in CI later.